### PR TITLE
I tried to repeat the example from video lecture2

### PR DESCRIPTION
**Pull request by AleksandrBohan**

### CODE REVIEW

**Review by BulavaKatsyaryna — APPROVED** [review]

**Review by yurasava — APPROVED** [review]

**Comment by KaydunovDenis** [comment]
Я не уверен, что это стоит мержить в мастерскую ветвь.

**Comment by KaydunovDenis** [comment]
Если что то просто хочешь попробовать попушить, покомитить, то я думаю лучше это делать в свой личный репозитории, а не в общий

File: lecturer/junit_intro/src/main/java/by/jrr/start/Calculator.java [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 
-    public int sum(int a, int b) {
-        return a + b;
+    public int multiplikation(int a, int b) {
+        return a * b;
     }
 
-    public int subs(int a, int b) {
-        return a - b;
+    public int division(int a, int b) {
```
Comment (max-java) [inline lecturer/junit_intro/src/main/java/by/jrr/start/Calculator.java:11]: should perform only in b != 0;

File: lecturer/junit_intro/src/main/java/by/jrr/start/Calculator.java [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
+        return a/b;
```
Comment (max-java) [inline lecturer/junit_intro/src/main/java/by/jrr/start/Calculator.java:12]: use sonar analyse.
// a/b 
this need to be surrounded with spaces.

File: lecturer/junit_intro/src/test/java/by/jrr/start/ApplicationTest.java [hunk 3]
```diff
@@ -10,10 +10,13 @@
     @Test
     public void main() {
 
-        int expected = 1;
+        int expected = 2;
 
         int actual = Integer.valueOf("2");
 
         Assert.assertEquals("Failed", expected, actual);
```
Comment (max-java) [inline lecturer/junit_intro/src/test/java/by/jrr/start/ApplicationTest.java:17]: Assert.assertEquals("Failed");
  Use meaningful messages.

File: lecturer/junit_intro/src/test/java/by/jrr/start/ApplicationTest.java [hunk 4]
```diff
@@ -10,10 +10,13 @@
...
     }
+
+    public void testTestMain() {
```
Comment (max-java) [inline lecturer/junit_intro/src/test/java/by/jrr/start/ApplicationTest.java:20]: should be deleted

File: lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java [hunk 5]
```diff
@@ -33,55 +33,46 @@ public static void GlobalTearDown() {
...
-    public void sum() {
-        int a = 4;
-        int b = 3;
-
-        int expected = 7;
-        int actual = calculator.sum(a, b);
-
+    public void testDivision() {
+        int a = 12;
+        int b = 4;
```
Comment (max-java) [inline lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java:38]: try with b= 0,

File: lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java [hunk 6]
```diff
@@ -33,55 +33,46 @@ public static void GlobalTearDown() {
...
+        int expected = 3;
+        int actual = calculator.division(a,b);
```
Comment (max-java) [inline lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java:40]: use sonar, spaces should be added

File: lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java [hunk 7]
```diff
@@ -33,55 +33,46 @@ public static void GlobalTearDown() {
...
-        int b = 3;
-
-        int expected = 1;
-        int actual = calculator.subs(a, b);
-
+    public void testMultiplikation() {
+        int a = 3;
+        int b = 4;
+        int expected = 12;
+        int actual = calculator.multiplikation(a,b);
```
Comment (max-java) [inline lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java:49]: the same as above

File: lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java [hunk 8]
```diff
@@ -33,55 +33,46 @@ public static void GlobalTearDown() {
...
-
-        int expected = -4;
-        calculator.setResult(calculator.subs(a, b));
-        calculator.setResult(calculator.sum(calculator.getResult(), c));
-
+        int c = 2;
+        int expected = 0;
         int actual = calculator.getResult();
-
         assertEquals(expected, actual);
```
Comment (max-java) [inline lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java:72]: assertions should be in the end of the test

File: lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java [hunk 9]
```diff
@@ -33,55 +33,46 @@ public static void GlobalTearDown() {
...
+        int a = 6;
         int b = 3;
-        int c = -5;
-
-        int expected = -4;
-        calculator.setResult(calculator.subs(a, b));
-        calculator.setResult(calculator.sum(calculator.getResult(), c));
-
+        int c = 2;
+        int expected = 0;
```
Comment (max-java) [inline lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java:70]: defiantly expected not 0!

**Review by max-java — CHANGES_REQUESTED** [review]
please move this to your homework folder
left comments.

File: lecturer/junit_intro/src/main/java/by/jrr/start/Calculator.java (hunk 1 above)
Comment (AleksandrBohan) [reply lecturer/junit_intro/src/main/java/by/jrr/start/Calculator.java:11]: I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files

File: lecturer/junit_intro/src/test/java/by/jrr/start/ApplicationTest.java (hunk 3 above)
Comment (AleksandrBohan) [reply lecturer/junit_intro/src/test/java/by/jrr/start/ApplicationTest.java:17]: I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.

File: lecturer/junit_intro/src/test/java/by/jrr/start/ApplicationTest.java (hunk 4 above)
Comment (AleksandrBohan) [reply lecturer/junit_intro/src/test/java/by/jrr/start/ApplicationTest.java:20]: I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.

File: lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java (hunk 5 above)
Comment (AleksandrBohan) [reply lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java]: I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.

File: lecturer/junit_intro/src/main/java/by/jrr/start/Calculator.java (hunk 2 above)
Comment (AleksandrBohan) [reply lecturer/junit_intro/src/main/java/by/jrr/start/Calculator.java]: Sonar tests failed.

File: lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java (hunk 6 above)
Comment (AleksandrBohan) [reply lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java]: I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.
I had added spaces, but sonar tests still failed. Help me, please.

File: lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java (hunk 7 above)
Comment (AleksandrBohan) [reply lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java]: I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.

File: lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java (hunk 8 above)
Comment (AleksandrBohan) [reply lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java]: I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.

File: lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java (hunk 9 above)
Comment (AleksandrBohan) [reply lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java]: I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.

**Pull request closed without merging**